### DISCUSSION TITLE
What happens if one of my favorite tracks removed from Youtube?

### DISCUSSION THREAD

**Discussion by JFMugen** [nukeop/nuclear#1055] [Q&A]
I hope it won't be removed from the list without any notice?

**Comment by nukeop** [comment]
It's saved locally as [artist] - [track]. The exact stream is searched for when you add it to the queue. It doesn't depend on the video on Youtube.

**Reply by JFMugen** [reply]
That's awesome! Thank you for the answer and the app :)
